charts/seed/crds/volumesnapshot.yaml

```diff
@@ -1,3 +1,4 @@
+{{- if and (not .Values.seedKubeadm) (not .Values.seedVirtual) }}
```

Comment:
shouldn't this be an `or` condition - otherwise the ceph clusters will get this? same goes for all other resources with `and` condition

Reply:
I mean it's currently seeded in the ceph clusters. But isn't the condition `false` if either one or both are `true`? Also see the updated description.